lib/basedriver/helpers.js

```diff
@@ -306,24 +306,40 @@ function getSwipeTouchDuration (waitGesture) {
 }
 
 /**
- * Recursively find all instances of the key 'inKey' and rename them 'outKey'
+ * Finds all instances 'firstKey' and create a duplicate with the key 'secondKey',
+ * Do the same thing in reverse. If we find 'secondKey', create a duplicate with the key 'firstKey'.
+ *
+ * This will cause keys to be overwritten if the object contains 'firstKey' and 'secondKey'.
+
  * @param {*} input Any type of input
- * @param {String} inKey The key name to replace
- * @param {String} outKey The key name to replace it with
+ * @param {String} firstKey The first key to duplicate
+ * @param {String} secondKey The second key to duplicate
  */
-function renameKey (input, inKey, outKey) {
+function duplicateKeys (input, firstKey, secondKey) {
+  // If array provided, recursively call on all elements
   if (_.isArray(input)) {
-    return input.map((item) => renameKey(item, inKey, outKey));
-  } else if (_.isPlainObject(input)) {
-    return _.reduce(input, (resultObj, value, key) => ({
-      ...resultObj,
-      [key === inKey ? outKey : key]: renameKey(value, inKey, outKey),
-    }), {});
+    return input.map((item) => duplicateKeys(item, firstKey, secondKey));
+  }
+
+  // If object, create duplicates for keys and then recursively call on values
+  if (_.isPlainObject(input)) {
+    const resultObj = {};
+    for (let [key, value] of _.toPairs(input)) {
+      const recursivelyCalledValue = duplicateKeys(value, firstKey, secondKey);
+      if (key === firstKey) {
+        resultObj[secondKey] = recursivelyCalledValue;
+      } else if (key === secondKey) {
+        resultObj[firstKey] = recursivelyCalledValue;
+      }
+      resultObj[key] = recursivelyCalledValue;
```

Comment:
else?

Reply:
No. This always needs to be done. Every key needs to be copied over.